transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/UpdateTransferMethodPresenter.java

```diff
@@ -0,0 +1,110 @@
+package com.hyperwallet.android.ui.transfermethod.view;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.hyperwallet.android.model.Error;
+import com.hyperwallet.android.model.Errors;
+import com.hyperwallet.android.model.graphql.HyperwalletTransferMethodConfigurationField;
+import com.hyperwallet.android.model.transfermethod.TransferMethod;
+import com.hyperwallet.android.ui.R;
+import com.hyperwallet.android.ui.transfermethod.repository.TransferMethodRepository;
+import com.hyperwallet.android.ui.transfermethod.repository.TransferMethodUpdateConfigurationRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class UpdateTransferMethodPresenter implements UpdateTransferMethodContract.Presenter {
+    private static final String ERROR_UNMAPPED_FIELD = "ERROR_UNMAPPED_FIELD";
+    private final TransferMethodUpdateConfigurationRepository mTransferMethodUpdateConfigurationRepository;
+    private final TransferMethodRepository mTransferMethodRepository;
+    private final UpdateTransferMethodContract.View mView;
+
+    public UpdateTransferMethodPresenter(UpdateTransferMethodContract.View view,
+            TransferMethodUpdateConfigurationRepository transferMethodUpdateConfigurationRepository,
+            TransferMethodRepository transferMethodRepository) {
+        mView = view;
+        mTransferMethodUpdateConfigurationRepository = transferMethodUpdateConfigurationRepository;
+        mTransferMethodRepository = transferMethodRepository;
+    }
+
+    @Override
+    public void loadTransferMethodConfigurationFields(boolean forceUpdate, @NonNull String transferMethodType,
+            @NonNull final String transferMethodToken) {
+        mView.showProgressBar();
+
+        if (forceUpdate) {
+            mTransferMethodUpdateConfigurationRepository.refreshFields();
+        }
+
+        mTransferMethodUpdateConfigurationRepository.getFields(transferMethodType,
+                transferMethodToken,
+                new TransferMethodUpdateConfigurationRepository.LoadFieldsCallback() {
+                    @Override
+                    public void onFieldsLoaded(@Nullable HyperwalletTransferMethodConfigurationField field) {
+                        if (!mView.isActive()) {
+                            return;
+                        }
+
+                        mView.hideProgressBar();
+                        mView.showTransferMethodFields(field);
+                        // there can be multiple fees when we have flat fee + percentage fees
+                        mView.showTransactionInformation(field.getFees(), field.getProcessingTime());
```

Comment:
same here, no need for fees/processing time in edit transfer method

Reply:
removed

Reply:
removed